optaplanner-core/src/test/java/org/optaplanner/core/impl/localsearch/decider/acceptor/hillclimbing/HillClimbingAcceptorTest.java

```diff
@@ -29,20 +29,20 @@
 public class HillClimbingAcceptorTest extends AbstractAcceptorTest {
 
     @Test
-    public void hillClimbingEnabled() {
+    public <Solution_> void hillClimbingEnabled() {
         HillClimbingAcceptor acceptor = new HillClimbingAcceptor();
 
-        DefaultSolverScope solverScope = new DefaultSolverScope();
+        DefaultSolverScope<Solution_> solverScope = new DefaultSolverScope<>();
```

Comment:
+0. The exact type parameter has no effect. Even `Object` works. So I went with an unspecified parameter. And `Solution_` is shorter than `TestdataSolution`.

I'll change that as you request. I suppose that all tests should be updated.